src/javascripts/templates/templateTransaction.js

```diff
@@ -1,13 +1,13 @@
-import { formatDate, formatNumber } from "../helpers/helpers"
+import { formatDate, formatNumber } from "../helpers/format"
 
-class Template {
+class TemplateTransaction {
```

Comment:
Should be TransactionTemplate

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks !